controls/radwizard/getting-started.md

```diff
@@ -19,37 +19,56 @@ This tutorial will walk you through the creation of a sample application that co
 >
 >You can find more info {% if site.site_name == 'Silverlight' %}[here]({%slug installation-installing-controls-dependencies%}){% endif %}{% if site.site_name == 'WPF' %}[here]({%slug installation-installing-controls-dependencies-wpf%}){% endif %}  .
 
-For the purpose of this example, you will need to open Visual Studio and then create an empty Application project.
-
 ## Adding RadWizard to the Project
-You can add __RadWizard__ manually by writing the XAML code in __Example 1__. You can also add the control by dragging it from the Toolbox. 
+You can add __RadWizard__ in XAML or in code. You can also add the control by dragging it from the [Toolbox]({%slug installation-adding-to-vs-2017-toolbox%}). 
```

Comment:
Leave a blank new line between the header of the section and its content.

Header

Some Content here